src/types/cron.types.ts

```diff
@@ -30,7 +30,9 @@ export type CronJobParams<
 				timeZone?: never;
 				utcOffset?: number | null;
 		  }
-	);
+	) & {
+		errorHandler?: (error: unknown) => void;
```

Comment:
this should go in the `BaseCronJobParams`

Reply:
also it should allow both `undefined` and `null` for quality of life (like the other params), so it should be something like:
```ts
errorHandler?: ((error: unknown) => void) | null;
```

Reply:
Moved it to `BaseCronJobParams` and allowed `null`